### DISCUSSION TITLE
Top language issue

### DISCUSSION THREAD

**Discussion by YannickBOYER** [anuraghazra/github-readme-stats#1780] [Q&A] [answered]
Does anyone know why I can't see C# on my top languages ? It's supposed to be my second language I think
https://github-readme-stats.vercel.app/api/top-langs/?username=Yannick2510

**Comment by Rongronggg9** [comment, marked as answer]
https://github.com/anuraghazra/github-readme-stats/blob/de05a574b145ffa6d2813c4fa53bc4fca67cb732/src/fetchers/top-languages-fetcher.js#L14-L32

The fetcher only fetches you-owned-non-fork repositories.

**Reply by ChristmasCatt** [reply]
👍

**Comment by elenakrittik** [comment]
> https://github.com/anuraghazra/github-readme-stats/blob/de05a574b145ffa6d2813c4fa53bc4fca67cb732/src/fetchers/top-languages-fetcher.js#L14-L32
> 
> The fetcher only fetches you-owned-non-fork repositories.

But look at [my card](https://github.com/weerdy15). *The only repository i own which contains javascript is a fork* of this repo.

**Comment by elenakrittik** [comment]
My case was a cache issue 😅

**Reply by ChristmasCatt** [reply]
👍

**Comment by haideralicodes** [comment]
query userInfo($login: String!) { 
         user(login: $login) { 
           # fetch only owner repos & not forks 
           repositories(ownerAffiliations: OWNER, isFork: false, first: 100) { 
             nodes { 
               name 
               languages(first: 10, orderBy: {field: SIZE, direction: DESC}) { 
                 edges { 
                   size 
                   node { 
                     color 
                     name 
                   } 
                 } 
               } 
             } 
           } 
         } 
       }

**Comment by ChristmasCatt** [comment]
👍

**Comment by ChristmasCatt** [comment]
👌

**Comment by alirezachali** [comment]
Hey. Dude. That was cool.
